animation.js

```diff
@@ -117,6 +234,7 @@ Animation.prototype.update = function () {
   })
 }
 
+
```

Comment:
Try to remove extra space when committing, it makes the process of reviewing leaner.